plugins/common/opcua/opcua_util_test.go

```diff
@@ -0,0 +1,118 @@
+package opcua
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// fileExists checks if a file exists at the given path (test helper)
+func fileExists(path string) bool {
+	_, err := os.Stat(path)
+	return err == nil
+}
```

Comment:
Please use [`require.FileExists()`](https://pkg.go.dev/github.com/stretchr/testify/require#FileExists) and the `NoFileExists` counterpart instead of implementing this function!